mods/gen4/random-teams.js

```diff
@@ -61,6 +61,14 @@ class RandomGen4Teams extends RandomGen5Teams {
 			'Adaptability':1, 'Hustle':1, 'Iron Fist':1, 'Skill Link':1,
 		};
 
+		// Give recovery moves priority over certain other defensive status moves
+		let recoveryMoves = {
+			'healorder':1, 'milkdrink':1, 'moonlight':1, 'morningsun':1, 'recover':1, 'rest':1, 'roost':1, 'slackoff':1, 'softboiled':1, 'synthesis':1, 'wish':1,
```

Comment:
I'm not necessarily hoping to get Rest + Sleep Talk (or Wish + Protect) when I reject the status move though. I'm just rejecting the status move because it's almost certainly not a good move on these Pokemon if their set doesn't include recovery. For example if I have a Snorlax with 3 attacks + Whirlwind, rejecting Whirlwind will most likely turn it into CB 4 attacks or Curse + 3 attacks, both of which are much better, but it still leaves room for Whirlwind to show up on a RestTalk set.